WordPress/src/main/java/org/wordpress/android/ui/jetpackplugininstall/fullplugin/onboarding/compose/state/LoadedState.kt

```diff
@@ -33,7 +33,7 @@ import org.wordpress.android.ui.jetpackplugininstall.fullplugin.onboarding.compo
 import org.wordpress.android.ui.jetpackplugininstall.fullplugin.onboarding.compose.component.TermsAndConditions
 
 @Composable
-fun LoadedState(
+fun loadedState(
```

Comment:
Awesome @thomashorta and thank you for looking into that in more detail! 🥇 

> opened an [issue for Google](https://issuetracker.google.com/u/1/issues/283369694) and hopefully, this will be fixed in future releases.

This is great, thank you for doing so! 💯 

> In the meantime, I believe we could use capitalized function names and explicitly declare the return type of Unit. What do you think?

I agree, let me go and update those function names back to uppercase, just like you are suggesting! 👍

Reply:
Wow, great find @thomashorta 🙇🏻 , TIL for me! I can only agree then, to restore the Capitals!